### PR TITLE
UserGuide: Added introduction

### PR DESCRIPTION
**Pull request by zacharytang**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -14,6 +14,18 @@ endif::[]
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
 
+== Introduction
+Thank you for downloading **UniFy**! We're glad that you've chosen us to help you manage your hectic university life.
```
Comment (April0616) [inline docs/UserGuide.adoc:18]: interesting

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -14,6 +14,18 @@ endif::[]
...
+
+**UniFy** is a personal assistant to your everyday life in NUS. It helps you maintain your numerous groups,
+tells you when your friends are free for lunch and find common time for project meetings. It even helps you find directions to
+wherever your friends might be!
```
Comment (April0616) [inline docs/UserGuide.adoc:22]: 'wherever your friends might be'? Is it related to the GPS thing which we don't intend to implement?   :)

**Review by April0616 — APPROVED** [review]
Generally good!

File: docs/UserGuide.adoc (hunk 2 above)
Comment (zacharytang) [reply docs/UserGuide.adoc:22]: This refers to their address which is already implemented in the Person field :)

**Pull request merged**